rs/bitcoin/kyt/src/blocklist.rs

```diff
@@ -1,344 +1,439 @@
-/// The list of addresses to which we do not allow retrievals.
+/// Bitcoins are not accepted from nor sent to addresses on this list.
```

Comment:
It would maybe be useful to explain from which source this list was retrieved from and how it was generated.

Reply:
I agree. I'm in the process of unifying this for Bitcoin and Ethereum. How about we do this in a separate PR?

Reply:
Separate PR also sounds good!